webapp/src/client/client.js

```diff
@@ -15,7 +15,7 @@ export default class Client {
         headers['X-Requested-With'] = 'XMLHttpRequest';
         headers['X-Timezone-Offset'] = new Date().getTimezoneOffset();
 
-        try {
+        try { // eslint-disable-line no-useless-catch
```

Comment:
Why not drop all the useless catches? That's what I did with other dependency updates.

Reply:
Good point, done.